src/pages/WatchList.js

```diff
@@ -152,30 +152,38 @@ function WatchList() {
             <Tab>All</Tab>
           </TabList>
           <TabPanels>
-            <TabPanel>
-              {/* Unwatched items */}
-              {listMovies
-                .filter((movie) => !movie.watched)
-                .map((unwatchedMovie) => (
-                  <ListItem
-                    key={unwatchedMovie.id}
-                    data={unwatchedMovie}
-                    listDetails={listDetails}
-                  />
-                ))}
-            </TabPanel>
+            {!listMovies || listMovies.length === 0 ? (
+              findPrompt()
```

Comment:
I like the idea here.  maybe instead of having to check on each tab, render either the find movies component or if there are movies render the whole tab group? your call, just a thought...

Reply:
Lol that does make a lot more sense. I'll implement that.